store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/util/MergeElementFunction.java

```diff
@@ -56,16 +70,16 @@ public class ApplyViewToElementsFunction implements ContextSpecificMergeFunction
     @JsonProperty("context")
     private Map<String, Object> context;
 
-    public ApplyViewToElementsFunction() {
+    public MergeElementFunction() {
     }
 
-    public ApplyViewToElementsFunction(final Map<String, Object> context) throws GafferCheckedException {
+    public MergeElementFunction(final Map<String, Object> context) throws GafferCheckedException {
```

Comment:
Can this be refactored? The method is doing more than one thing. Can we have separate methods for each step of the process?

Reply:
Yes, the constructor is doing far too much here. But this code pre-dated this PR and branch.
I will be happy to for us to go through a refactor session together over this.

Reply:
please examine https://github.com/gchq/Gaffer/pull/3165